man/man8/tc.8

```diff
@@ -445,7 +445,8 @@ Bandwidths or rates.
 These parameters accept a floating point number, possibly followed by
 either a unit (both SI and IEC units supported), or a float followed '%'
```

Comment:
This is not from this patch, but from the patch before: should be "followed **by a** '%'"